core/src/main/scala/org/apache/spark/status/LiveEntity.scala

```diff
@@ -325,6 +325,9 @@ private class LiveExecutorStageSummary(
 
   var metrics = createMetrics(default = 0L)
 
+  // peak values for executor level metrics
+  var peakExecutorMetrics = new ExecutorMetrics()
```

Comment:
can this also be a val like the one in line 387?

Reply:
Yes, this should be val.